jetty-start/src/main/java/org/eclipse/jetty/start/Main.java

```diff
@@ -470,8 +481,7 @@ else if (args.isCreateFiles() || !args.getStartModules().isEmpty())
             CommandLineBuilder cmd = args.getMainArgs(StartArgs.ALL_PARTS);
             cmd.debug();
 
-            List<String> execModules = args.getEnabledModules().stream()
-                .map(name -> args.getAllModules().get(name))
+            List<String> execModules = args.getAllModules().getEnabled().stream()
```

Comment:
How is `args.getAllModules().getEnabled().stream()` different from `args.getEnabledModules().stream()`?

If it is different, then we have a naming problem.  If it is not different then just use `getEnabledModules()`

Reply:
I don't see this as a naming difference, just a "source of truth" difference.   
The `Modules` vs `StartArgs` are difference classes, with different purposes.